lib/AST/ASTDumper.cpp

```diff
@@ -1347,6 +1347,14 @@ namespace {
       printStorageImpl(VD);
       printFlag(VD->getAttrs().hasAttribute<KnownToBeLocalAttr>(),
                 "known_to_be_local", DeclModifierColor);
+      if (auto *nonisolatedAttr =
+              VD->getAttrs().getAttribute<NonisolatedAttr>()) {
```

Comment:
IMO a better approach would be to print the result of `getActorIsolation` for all declarations